django/utils/log.py

```diff
@@ -0,0 +1,16 @@
+from fluent.handler import FluentRecordFormatter
+
+
+class FluentFormatter(FluentRecordFormatter):
+    def format(self, record):
+        """
+        Format the record as dict.
+
+        If there is exception information, it is formatted using formatException() and added to the data.
+        """
+        data = super().format(record)
+
+        if record.exc_info:
+            data["traceback"] = self.formatException(record.exc_info)
```

Comment:
Oh, this answers my comment about logging KafkaExceptions, right?

Reply:
Calling `logger.exception()` automatically adds `exc_info` to the log record. We use this to add the formatted traceback to the `data`. The `FluentRecordFormatter` adds the `data` to the formatted msg.

I don't know if its answer your comment about KafkaException